modules/ROOT/pages/managing-users.adoc

```diff
@@ -90,9 +90,19 @@ If enabled, the SAML assertions sent from the IdP must be encrypted and follow t
 +
 . Sign out of Anypoint Platform, navigate to the sign-on URL you entered in the *Identity Management SAML 2.0* form, then sign in through your identity provider to test the configuration.
 
+== Import SAML Metadata
+
+You can import your identity provider's SAML 2.0 metadata to Anypoint Platform using the *Import IdP Metadata* link. Once uploaded, the XML file's values automatically populate the *Identity Management SAML 2.0* form. Note that you can import the SAML 2.0 metadata only when you are creating a new SAML 2.0 configuration.
+
+When you upload an XML file to import your metadata:
```

Comment:
So...the upload automatically populates the form; all that "you" do is click (?) the link to import.
YOU are IMPORTING; the process is UPLOADING and POPULATING, right?
This sentence doesn't exactly reflect what is in the para above it.

Reply:
Again, should we make this a task with an intro saying you can do this and WHY, mentioning the prerequisite file size limit and new config requirements, having a couple of steps, and then following with the second two of the three following items?

Reply:
Should it really be a task if it's only one step? They import, and the rest is done for them.